Makefile

```diff
@@ -0,0 +1,55 @@
+# -------------------------------------------------------------
+# This makefile defines the following targets
+#   - tape - builds a binary program
+#   - tape-docker - build the tape image
+#   - unit-test - runs the go-test based unit tests
+#   - integration-test - runs the integration tests
+
+FABRIC_VERSION = latest
+INTERGATION_CASE = ANDLogic
+
+BASE_VERSION = 0.0.2 
+PREV_VERSION = 0.0.1
+
+PROJECT_NAME = tape
+DOCKERIMAGE = guoger/tape
+export DOCKERIMAGE
+EXTRA_VERSION ?= $(shell git rev-parse --short HEAD)
+BuiltTime ?= $(shell date)
+PROJECT_VERSION=$(BASE_APISERVER_VERSION)-snapshot-$(EXTRA_VERSION)
+
+PKGNAME = github.com/guoger/$(PROJECT_NAME)
+CGO_FLAGS = CGO_CFLAGS=" "
+ARCH=$(shell go env GOARCH)
+MARCH=$(shell go env GOOS)-$(shell go env GOARCH)
+
+# defined in pkg/infra/version.go
+METADATA_VAR = Version=$(BASE_VERSION)
+METADATA_VAR += CommitSHA=$(EXTRA_VERSION)
+
+GO_LDFLAGS = $(patsubst %,-X $(PROJECT_NAME)/pkg/infra.%,$(METADATA_VAR))
+GO_LDFLAGS += -X '$(PROJECT_NAME)/pkg/infra.BuiltTime=$(BuiltTime)'
+
+GO_TAGS ?=
+
+export GO_LDFLAGS GO_TAGS FABRIC_VERSION INTERGATION_CASE
+
+.PHONY: tape
+tape:
+	@echo "Building tape program......"

```

Comment:
question, can we use `make tape` without given `GO_TAGS` and `GO_LDFLAGS`?

Reply:
Currently GO_TAGS is empty, which can be deleted. 
GO_LDFLAGS assigns Version, CommitSHA and BuiltTime in program.